### PR TITLE
chore: default flat runs to on

### PR DESCRIPTION
**Pull request by ashtonG**
## Ticket
ET-636

## Description
This defaults the flat runs view to on while also adding release notes

## Test Plan
* clear your settings
* go to the uncategorized project page
* you should see the flat runs view

## Checklist

- [x] Changes have been manually QA'd
- [x] New features have been approved by the corresponding PM
- [x] User-facing API changes have the "User-facing API Change" label
- [x] Release notes have been added as a separate file under `docs/release-notes/`
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [x] Licenses have been included for new code which was copied and/or modified from any external code

[DET-123]: https://hpe-aiatscale.atlassian.net/browse/DET-123?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/release-notes/run-centric-ux.rst [hunk 1]
```diff
@@ -0,0 +1,24 @@
+:orphan:
+
+**New Features**
+
+-  WebUI: Turn on new run-centric search view
```
Comment (tara-det-ai) [inline docs/release-notes/run-centric-ux.rst:5]:
```suggestion
- WebUI: In the Experimental features, Flat Runs View is now "on" by default. Users can still toggle this feature "off".
```

File: docs/release-notes/run-centric-ux.rst [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+   -  This replaces the existing experiment search and multi-trial experiment views with views that
```
Comment (tara-det-ai) [inline docs/release-notes/run-centric-ux.rst:7]:
```suggestion
- This update improves the ability to compare model performance between different trials, based on user feedback that most Determined users run single-trial experiments.
```

File: docs/release-notes/run-centric-ux.rst [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+      allow comparison between arbitrary trials.
```
Comment (tara-det-ai) [inline docs/release-notes/run-centric-ux.rst:8]:
```suggestion
```

File: docs/release-notes/run-centric-ux.rst [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+   -  We recieved user feedback indicating that users of Determined run primarily single-trial
```
Comment (tara-det-ai) [inline docs/release-notes/run-centric-ux.rst:10]:
```suggestion
```

File: docs/release-notes/run-centric-ux.rst [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+      experiments, and the preexisting views made comparing model performance between different
```
Comment (tara-det-ai) [inline docs/release-notes/run-centric-ux.rst:11]:
```suggestion
```

File: docs/release-notes/run-centric-ux.rst [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+      trials difficult.
```
Comment (tara-det-ai) [inline docs/release-notes/run-centric-ux.rst:12]:
```suggestion
```

File: docs/release-notes/run-centric-ux.rst [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+   -  We are renaming experiments to searches and trials to runs to make the difference between the
```
Comment (tara-det-ai) [inline docs/release-notes/run-centric-ux.rst:14]:
```suggestion
- "Experiments" is now renamed to "searches" and "trials" is now renamed to "runs" for better clarity.
```

File: docs/release-notes/run-centric-ux.rst [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+      two easier to understand.
```
Comment (tara-det-ai) [inline docs/release-notes/run-centric-ux.rst:15]:
```suggestion
```

File: docs/release-notes/run-centric-ux.rst [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+
+   -  The experiment list is now the run list, which shows all trials from experiments in the
```
Comment (tara-det-ai) [inline docs/release-notes/run-centric-ux.rst:17]:
```suggestion
- The "experiment list" is renamed the "run list", showing all trials from experiments in the
```

File: docs/release-notes/run-centric-ux.rst [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+      project. It should function similar to the previous new experiment list.
```
Comment (tara-det-ai) [inline docs/release-notes/run-centric-ux.rst:18]:
```suggestion
project. It functions similarly to the previous new experiment list.
```

File: docs/release-notes/run-centric-ux.rst [hunk 11]
```diff
@@ -0,0 +1,24 @@
...
+
+   -  Multi-trial experiments can be viewed in the new searches view, which allows for sorting,
+      filtering and navigation of the multi-trial experiments in the project.
```
Comment (tara-det-ai) [inline docs/release-notes/run-centric-ux.rst:21]:
```
    filtering and navigating multi-trial experiments.
```

File: docs/release-notes/run-centric-ux.rst [hunk 12]
```diff
@@ -0,0 +1,24 @@
...
+
+   -  When viewing a multi-trial experiment, we now show a list of trials in the experiment to allow
```
Comment (tara-det-ai) [inline docs/release-notes/run-centric-ux.rst:23]:
```suggestion
- When viewing a multi-trial experiment, a list of trials is displayed, allowing
```

File: docs/release-notes/run-centric-ux.rst [hunk 13]
```diff
@@ -0,0 +1,24 @@
...
+      for sorting, filtering and arbitrary comparison.
```
Comment (tara-det-ai) [inline docs/release-notes/run-centric-ux.rst:24]:
```suggestion
      for sorting, filtering and arbitrary comparison between trials.
```

**Review by tara-det-ai — COMMENTED** [review]
suggeste edits

File: webui/react/src/components/ProjectCard.test.tsx [hunk 14]
```diff
@@ -53,7 +53,7 @@ describe('ProjectCard', () => {
   it('should display experiments count', () => {
     setup();
     expect(
-      screen.getByText(projectMock.numExperiments?.toString() ?? 'Count undefined'),
+      screen.getByText(projectMock.numRuns?.toString() ?? 'Count undefined'),
```
Comment (johnkim-det) [inline webui/react/src/components/ProjectCard.test.tsx:56]: also change in `projectMock`

**Review by johnkim-det — COMMENTED** [review]
I think unit tests are good after the fix in comment, but I'm realizing the e2e tests for experiments may fail. perhaps we need to skip those in this PR?

**Review by johnkim-det — APPROVED** [review]
approving assuming e2e test issues are resolved

**Comment by ashtonG** [comment]
@johnkim-det E2E and unit tests are green

**Review by JComins000 — APPROVED** [review]
looks great as long as it's working. thank you for making the change!

**Pull request merged**